assets/javascripts/discourse/components/chat-live-pane.js

```diff
@@ -618,6 +618,16 @@ export default Component.extend({
           stagedId: null,
           rateLimited: false,
         });
+
+        // some markdown is cooked differently on the server-side, e.g.
+        // quotes, avatar images etc.
+        if (
+          data.chat_message.cooked &&
+          data.chat_message.cooked !== stagedMessage.cooked
+        ) {
```

Comment:
```suggestion
        if (data.chat_message.cooked !== stagedMessage.cooked) {
```

Reply:
This won't work, sometimes it is undefined or null for whatever reason! :upside_down_face:

Reply:
👀 I'd like us to know the reason 😅